Classes/Core/Functional/FunctionalTestCase.php

```diff
@@ -358,6 +359,20 @@ protected function setUpBackendUserFromFixture($userUid)
     {
         $this->importDataSet($this->backendUserFixture);
 
+        return $this->setUpBackendUser($userUid);
+    }
+
+    /**
+     * Sets up Backend User which is already available in db
+     *
+     * @param int $userUid
+     * @return BackendUserAuthentication
+     * @throws Exception
+     */
+    protected function setUpBackendUser($userUid): BackendUserAuthentication
+    {
+        $this->importDataSet($this->backendUserFixture);
```

Comment:
Probably we should ensure the fixture file is just imported once (on calling e.g. `setUpBackendUser` multiple times) like e.g.

```
$this->setUpBackendUserFromFixture(1);
// ... import all the records as admin ...
$this->setBackendUserFromRecord(2);
// ... continue as editor ...
```

* `setUpBackendUserFromFixture` would load the fixture file and call `setBackendUserFromRecord`
* `setBackendUserFromRecord` would just do the database handling, but no additional import

Reply:
ah right, sorry, a copy paste error. will fix it right away